packages/core/src/config.ts

```diff
@@ -748,6 +748,7 @@ const composeEntryConfig = async (
     // Turn entries in array into each separate entry.
     const globEntryFiles = await glob(entryFiles, {
       cwd: root,
+      absolute: true,
```

Comment:
This will make the entry config of Rspack turn to be an absolute path. Although all the test passes. I'm not sure will it introduce any issues? 🤔

Reply:
Rspack supports absolute entry path, see https://rspack.dev/config/entry#path-type

I also found out that the `calcLongestCommonPath` method only accepts absolute paths,  which led me to use the `absolute: true` option.

![image](https://github.com/user-attachments/assets/34c18793-1fbc-4d0a-8007-19da2fa5319a)

Reply:
Got it!